loleaflet/src/map/Clipboard.js

```diff
@@ -479,7 +479,7 @@ L.Clipboard = L.Class.extend({
 		}
 
 		var text = this._getHtmlForClipboard();
-//		this._stopHideDownload(); - this confuses the browser ruins copy/cut on iOS
+		//		this._stopHideDownload(); - this confuses the browser ruins copy/cut on iOS
```

Comment:
@pelinKuran Do you see the oddness on this line? :)

That's not your fault. eslint does that. Please check this commit to see how this kind of commented-out lines are handled: https://github.com/CollaboraOnline/online/pull/313/commits/5a6bff875e345d7d7ec1c00f211109a5137c4b55

And do the same here manually. ;)